go/throttle/throttler.go

```diff
@@ -73,6 +76,9 @@ func NewThrottler(isLeaderFunc func() bool) *Throttler {
 		metricsHealth:          cache.New(cache.NoExpiration, 0),
 	}
 	throttler.ThrottleApp("abusing-app", time.Now().Add(time.Hour*24*365*10), DefaultThrottleRatio)
+	if memcacheServers := config.Settings().MemcacheServers; len(memcacheServers) > 0 {
+		throttler.memcacheClient = memcache.New(memcacheServers...)
```

Comment:
So, [this client uses round-robin over the list of servers provided](https://godoc.org/github.com/bradfitz/gomemcache/memcache#New), 👍